src/log-pcap.c

```diff
@@ -25,6 +25,10 @@
  * Pcap packet logging module.
  */
 
+#include <sys/types.h>
+#include <dirent.h>
```

Comment:
these will need checks in configure.ac and be included from suricata-common.h wrapped in HAVE_...H